site/content/en/docs/manual/advanced/organization.md

```diff
@@ -5,57 +5,128 @@ weight: 2
+  - [Switching between organizations](#switching-between-organizations)
+- [Organization page](#organization-page)
+  - [Invite members into organization](#invite-members-into-organization)
+  - [Delete organization](#delete-organization)
+
 ## Personal workspace
 
-Your `Personal workspace` will display the tasks and projects you've created.
+The account's default state is activated when no **Organization** is selected.
+
+It means that a personal account is used.
+
+![](/images/personal_account.jpg)
+
+## Create new organization
+
+To create an organization, do the following:
+
+1. Log in to the CVAT.
+2. On the top menu, click your **Username** > **Organization** > **+ Create**.
+
+   ![](/images/image233.jpg)
+
+3. Fill in the following fields and click **Submit**.
+
+   ![](/images/image234.jpg)
+
+<!--lint disable maximum-line-length-->
+
+| Field            | Description                                                         |
+| ---------------- | ------------------------------------------------------------------- |
+| **Short name**   | A name of the organization that will be displayed in the CVAT menu. |
+| **Full Name**    | Optional. Full name of the organization.                            |
+| **Description**  | Optional. Description of organization.                              |
+| **Email**        | Optional. Your email.                                               |
+| **Phone number** | Optional. Your phone number.                                        |
+| **Location**     | Optional. Organization address.                                     |
 
-## Create a new organization
+<!--lint enable maximum-line-length-->
 
-To create an organization, open the user menu, go to `Organization` and click `Create`.
+The created organization will be available
+at you **nickname** > **Organization**
 
-![](/images/image233.jpg)
+### Switching between organizations
 
-Fill in the required information to create your organization.
-You need to enter a `Short name` of the organization, which will be displayed in the menu.
-You can specify other fields: `Full Name`, `Description` and the organization contacts.
-Of them will be visible on the organization settings page.
+If you have more than one **Organization**,
+it is possible to switch between these **Organizations** at any given time.
 
-![](/images/image234.jpg)
+Follow these steps:
+
+1. In the top menu, select your **Username** > **Organization**.
+2. From the drop-down menu, under the **Personal space** section,
+   choose the desired **Organization**.
+
+![](/images/image233_1.jpg)
 
 ## Organization page
 
-To go to the organization page, open the user menu, go to `Organization` and click `Settings`.
+**Organization page** is a place, where you can edit the **Organization** information
+and invite users to **Organization**.
+
+![](/images/orgpage.jpg)
+
+To go to the O**rganization page**, do the following:
+
+1. On the top menu, click your **Username** > **Organization**.
```

Comment:
Important mark, it works only when organization is activated. 
If it is not activated, there is not **Settings** button